src/Infrastructure/Services/Factories/ApiManagerHelperFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
May I suggest you require strict type checking throughout the application.

`<?php declare(strict_types=1);`